contracts/art-token/ArtToken.sol

```diff
@@ -90,6 +91,7 @@ contract ArtToken is IArtToken, ArtTokenBase, EIP712 {
                 BUY_PERMIT_TYPE_HASH,
                 params.tokenId,
                 keccak256(bytes(params.tokenURI)),
+                msg.sender,
```

Comment:
It may also be nice to use not the sender but the receiver. It could be useful in the future, for example, if we want to add some logic to buy the NFT for another user, or smth like that. What do you think about it @Vladimir-Beliy ?

Reply:
With such an approach a sender will pay and a sender can be anyone, the admin sets a receiver,  I am not sure about this logic, and I decided to keep it simple and not add an additional parameter to invoke the buy method.

Reply:
OK. Let's keep it simple. We can update logic anytime, if we will need it